File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.java
import java.util.*;
import java.io.*;

class Solution {
    public int[] solution(int[] array) {
        int[] answer = new int[2];
        int max = Integer.MIN_VALUE;
        int maxIndex = -1;
        for(int i = 0; i < array.length; i++) {
            if(array[i] > max)
            {
                max = array[i];
                maxIndex = i;
            }          
        }
        
        answer[0] = max;
        answer[1] = maxIndex;
        return answer;
    }
}

File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.cpp
#include <string>
#include <vector>
#include <climits>

using namespace std;

vector<int> solution(vector<int> array) {
    vector<int> answer;
    int max = INT_MIN;
    int maxIndex = -1;
        
    for(int i = 0; i < array.size(); i++) {
        if(array[i] > max) {
            max = array[i];
            maxIndex = i;
        }          
    }
        
    answer.push_back(max);
    answer.push_back(maxIndex);
    return answer;
}